task.go

```diff
@@ -1,5 +1,10 @@
 package onfleet
 
+type PaginatedTasks struct {
+	LastID string `json:"lastId"`
```

Comment:
change LastID to LastId and add omitempty
refer to to TeamTasks struct in team.go which also includes LastId.